content/en/blog/falco-on-gke/index.md

```diff
@@ -0,0 +1,198 @@
+
+On the Google Cloud side, GKE uses Container-Optimized OS (COS) as the default operating system for its worker node pools. COS is a security-enhanced operating system that limits access to certain parts of the underlying OS. Because of this security constraint, Falco cannot insert its kernel module to process system calls. However, COS does support eBPF, so that’s the option we’ll use (more specifically we’ll use the classic eBPF probe)
+
+
+## Installing Falco via the Google Cloud Marketplace
+
+Note: If you’d like to follow along, you’ll need to ensure your Google Cloud account has the appropriate permissions. 
+
+Installing Falco via the Google Cloud Marketplace is a pretty straightforward process. 
+
+
+
+* Log into your Google Cloud account, and ensure you have the required permissions to deploy a new GKE cluster or operate an existing one. \
+
+* Navigate to the Falco offering in the [Google Cloud Marketplace](https://console.cloud.google.com/marketplace/product/google/falco).
+
+    !["screen shot of Falco on Google Cloud Marketplace"](./images/marketplace.png "screenshot of Falco on Google Cloud Marketplace")
+
+
+* Click the **configure** button. 
+
+* From the next dialog you can choose the zone where your GKE cluster will run as well as the network and subnet on which it will run. For this walkthrough, the default values are fine. 
+
+* You then choose whether or not you’d like to deploy Falco onto a new GKE cluster or use an existing one. Be aware that if you click **Create New Cluster,** Google Cloud will immediately start deploying a new cluster. Also, any Autopilot clusters that you have in your project will be grayed out and cannot be selected.  
+
+* You can then choose which namespace Falco will run in. To keep things consistent with the rest of this blog, change it from default to `falco`. 
+
+* Again, to keep things consistent with the rest of the blog, change the app instance name to `falco`. 
+
+* Falco rules have different priority levels, you can choose the minimum priority level you’d like to run. The priority levels are ordered by severity, and typically the higher you make the minimum level, the fewer alerts you will receive (which helps to cut down on noise). For this example just leave it as `debug`.
+
+* Stackdriver is the old name for Google Cloud’s logging and monitoring suite. If you’d like to examine Falco’s metrics (not the actual alerts, but metrics on how Falco is performing) you can select that option. We won’t be covering that in this blog, so go ahead and leave it unchecked. 
+
+
+    ![Screenshot of the configured options](./images/configuration.png "Screenshot of the configured options")
+
+* Click `DEPLOY` to deploy Falco onto the target cluster. 
+
+    > If you choose to deploy a new cluster, you will need to wait until that finishes to click the `DEPLOY` button. 
```

Comment:
In the rendered page, this sentence is hanging by itself out of nowhere. It looks weird. Hence the suggestion.
```suggestion
* Click `DEPLOY` to deploy Falco onto the target cluster. (If you choose to deploy a new cluster, you will need to wait until that finishes to click the `DEPLOY` button.)
```